stdlib/nodejs/io.rb

```diff
@@ -1,3 +1,64 @@
+class IO
+
+  @__fs__ = node_require :fs
+  `var __fs__ = #{@__fs__}`
+
+  attr_reader :eof
+  attr_reader :lineno
+
+  def initialize
+    @eof = false
+    @lineno = 0
+  end
+
+  def read(length = nil)
```

Comment:
@elia `outbuf` optional argument is not supported because I don't really know how to handle all the cases